packages/js/product-editor/changelog.md

```diff
@@ -2,10 +2,88 @@
+-   Minor - Always fallback to the standard product template in case the given product type is variable [#43422]
+-   Minor - Create attribute control custom empty state [#44001]
+-   Minor - Create linked product list block [#43145]
+-   Minor - Create variation empty state when no variable attributes are asigned to the product [#44163]
+-   Minor - Expose metadata as object from useProductMetadata hook [#43860]
+-   Minor - Fix margin top for none first child nested sections [#43864]
+-   Minor - Introduce a product type selection within the new experience [#41823]
+-   Minor - Load/save the product template from/to the product meta data [#43120]
+-   Minor - Product Editor: Subsection block #43725 [#43725]
+-   Minor - Set the product template id also for unsupported product templates [#43341]
+-   Minor - track Linked Products feature events [#43828]
+-   Minor - Unlink downloads and downloadable product props so they can be managed separately [#44152]
+-   Patch - [Product Block Editor]: add CashRegister image component [#43127]
+-   Patch - [Product Block Editor]: Add Linked Product sections [#43013]
+-   Patch - [Product Block Editor]: add loading for the `Choose products for me` button [#43656]
+-   Patch - [Product Block Editor]: add plain and rich text mode to the Textarea field block [#44191]
+-   Patch - [Product Block Editor]: add Upsell advice [#43116]
+-   Patch - [Product Block Editor]: always show "loading" state when choosing suggested products [#44053]
+-   Patch - [Product Block Editor]: exclude the current product from the suggested ones to use in the Linked Product tab [#43662]
+-   Patch - [Product Block Editor]: implement `Choose products for me` button [#43520]
+-   Patch - [Product Block Editor]: Introduce AdviceCard component [#43082]
+-   Patch - [Product Block Editor]: Introduce ButtonWithDropdownMenuProps component [#43103]
+-   Patch - [Product Block Editor]: introduce note attribute to Label component [#44198]
+-   Patch - [Product Block Editor]: introduce ShoppingBags component [#43042]
+-   Patch - [Product Block Editor]: introduce TextArea field block [#44104]
+-   Patch - [Product Block Editor]: introduce `<SectionActions />` slot [#43540]
+-   Patch - [Product Block Editor]: render empty state for the Cross-sells section [#43143]
+-   Patch - [Product Block Editor]: use Label component to render Textarea field block label [#44222]
+-   Patch - [Product Block Editor]: use text-area block to handle the variation description field [#44151]
+-   Minor - Add isLoading to useProductMetadata hook [#44156]
+-   Patch - bump php version in packages/js/*/composer.json [#42020]
+-   Major [ **BREAKING CHANGE** ] - Load layout templates via the REST API. Note that layout template modifications must now be hooked up in an action that is called when REST API endpoints are handled, such as `rest_api_init`. [#43589]
```

Comment:
Not sure if we want to include this and increase the version to 2.0 because of this, if the breaking change is reliant on the WooCommerce version and not the package version.